### PR TITLE
Prevent connect `scroll_to_paragraph` multiple times to `class_desc`

### PR DESCRIPTION
**Pull request by jsjtxietian**
Fixes https://github.com/godotengine/godot/issues/92405

### CODE REVIEW

File: editor/editor_help.cpp [hunk 1]
```diff
@@ -2340,7 +2340,11 @@ void EditorHelp::_help_callback(const String &p_topic) {
 
 	if (class_desc->is_ready()) {
 		// call_deferred() is not enough.
+		if (class_desc->is_connected(SceneStringName(draw), callable_mp(class_desc, &RichTextLabel::scroll_to_paragraph))) {
+			class_desc->disconnect(SceneStringName(draw), callable_mp(class_desc, &RichTextLabel::scroll_to_paragraph));
```
Comment (Maran23) [inline editor/editor_help.cpp:2344]: I wonder if we really need to disconnect and reconnect again? Or maybe we can just do
`if !connected -> connect`

Comment (jsjtxietian) [reply editor/editor_help.cpp:2344]: If the `line` is not changed during multiple connect, then yes. I will take a look more closely.

Comment (akien-mga) [reply editor/editor_help.cpp:2344]: I missed the argument binding, indeed. Makes sense, then.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**